client/v2/internal/util/util.go

```diff
@@ -23,7 +23,6 @@ func DescriptorKebabName(descriptor protoreflect.Descriptor) string {
 }
 
 // DescriptorDocs returns the leading comments of the descriptor.
-// TODO this does not work, to fix.
```

Comment:
We need to fix this, which never worked it seems, so that we can use latest api module into release/v0.50 (https://github.com/cosmos/cosmos-sdk/pull/19376)

Reply:
https://github.com/cosmos/cosmos-proto/pull/131